packages/marko-web-inquiry/browser/default-form.vue

```diff
@@ -169,6 +186,14 @@ export default {
       type: String,
       required: true,
     },
+    gdprMessage: {
```

Comment:
There should be a boolean flag to turn on or off the GDPR functionality. By default, it should be off.